src/word_debt_bot/cogs/game_commands.py

```diff
@@ -94,21 +94,75 @@ async def info(
         """
         Check someone's current cranes and debt. Shows your own info if no name is given.
         """
-        if user:
-            # player = self.game.get_player_by_name(name, optional=True)
+        if user and isinstance(user, str):
+            name = user
+            player = self.game.get_player_by_display_name(user, optional=True)
+        elif user:
+            name = user.name
             player = self.game.get_player(str(user.id), optional=True)
         else:
+            name = ctx.author.name
             player = self.game.get_player(str(ctx.author.id), optional=True)
 
         if not player:
-            name = user.name if user else ctx.author.name
             await ctx.send(f"Player '{name}' not found! Are they registered?")
             return
 
         await ctx.send(
-            f"Info for {player.display_name}:\nDebt: {player.word_debt:,}\nCranes: {player.cranes:,}"
+            f"Info for <@{player.user_id}> \n"
+            f"Display Name: {player.display_name}\n"
+            f"Languages: {player.languages}\n"
+            f"Debt: {player.word_debt:,}\n"
+            f"Cranes: {player.cranes:,}\n",
+            allowed_mentions=discord.AllowedMentions.none(),
         )
 
+    @commands.command(name="set")
+    async def set_(
+        self,
+        ctx,
+        key: str = commands.parameter(description="The thing you want to set."),
+        value: str = commands.parameter(description="What you want to set it to."),
+    ):
+        """
+        Configure user settings. Available settings: 'name', 'languages'
+        """
+        if any([character in value for character in ["<", ">", "@"]]):
+            await ctx.send("💔 Invalid value! 😭")
+            return
+
+        if key.lower() == "name":
+            if len(value) > 32:
+                await ctx.send("🙅 Value too long! 😔")
+                return
+            self.game.set_player_display_name(str(ctx.author.id), value)
+            self.journal(
+                {
+                    "command": "set",
+                    "user": str(ctx.author.id),
+                    "key": key,
+                    "value": value,
+                }
+            )
+        elif key.lower() == "languages":
+            if len(value) > 16:
```

Comment:
Is a limit of 16 too strict for languages? Some people are learning a lot of them...

Reply:
I thought the idea was we'd be using flags or abbreviations and they'd be displayed in certain places (could add them to the leaderboard, for example), so having a bunch would be inconvenient

But I don't care that much and the Welsh flag is like length 9 so I'll bump it up

Reply:
We could chop it off where it's displayed anyway as well